units/github_status_api.go

```diff
@@ -171,6 +186,17 @@ func (j *githubStatusUpdateJob) fetch() (*message.GithubStatus, error) {
 		status.Context = "evergreen"
 		status.Description = "patch must be manually authorized"
 		status.State = message.GithubStateFailure
+	} else if j.UpdateType == githubUpdateTypePushToCommitQueue {
+		status.Context = "evergreen/commitqueue"
```

Comment:
Could you add this (and other case(s)) where it appears to a constant, so that when we change one we change them all?

Reply:
Done.